.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
please add one empty line to the end of the file.
Also you may need `*__pycache__/` as there are still files like this.